### ISSUE TITLE
Issues on mobile

### ISSUE THREAD

**Issue by jwahlin** [Strategy11/formidable-modal#4]
I haven't tested this yet, but it has been reported that it's not possible to scroll down in a modal on a mobile device. Also, it has been reported that clicking the X doesn't work correctly on a mobile device.
https://formidablepro.com/help-desk/scroll-bar-in-modal-popup/

**Comment by Crabcyborg** [comment]
I tried out v2.0 on my phone while logging in to see the admin bar, like described in the ticket Jonathan shared,

![image](https://github.com/Strategy11/formidable-modal/assets/9134515/39ab7b27-ce97-4cae-a588-afa9e2dd5750)

The button is kind of cut off and aligned in the wrong spot.

On v3.0.1 it looks a lot better to me,

![image](https://github.com/Strategy11/formidable-modal/assets/9134515/ac921a55-8dc2-49d8-ad8d-4afdbc12a1dc)

@jonathanenlared Could I get your help confirming that this issue is now closed as of the v3.0 release?

Thank you!
